ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -51,7 +51,7 @@ if [ "${COMPUTE_NODE_REPLICAS}" -le 0 ] || [ "${SIZE_VARIANT}" = "compact" ]; th
   workers=0
 fi
 master_type=null
-master_type_prefix=""
+master_type_prefix="Standard_D4"
```

Comment:
we do not need this change, right?

Reply:
Yes, we need, this is to fix the bug as the item 2 in the first comment. I'll doing a test without SIZE_VARIANT set in env.

Reply:
I think that is just a document issue, `ipi-conf-azure` is existing there for a loooong time, when master_type is `null` in install-config.yaml, installer would help select a default vm size to launch, so maybe in older version oc OCP, maybe the default vm size is small, but for the newer version, that changed, so the document get out of date.

Reply:
+1, and if you change `master_type_prefix` to `Standard_D4` as default, then master machine's instance type will always be set to `Standard_D4s_v3` ( for amd), which is less cpu/memory than default one installer used `Standard_D8s_v3`. 
In the future, if default instance type is changed again, we could never test the default value in installer.

Reply:
FYI, here is [installer PR](https://github.com/openshift/installer/pull/5519) to change control plane default instance type from D4s_v3 to D8s_v3.

Reply:
@jianlinliu @jinyunma I got you, I'll revert that change and update the document instead.